config/capacities.json

```diff
@@ -323,5 +323,21 @@
             "wind": 3,
             "solar": 532
         }
+    },
+    "US": {    
+        "capacity": {
+            "biomass": 16099,
+            "nuclear": 103860,
+            "coal": 304789,
+	        "gas" : 506760,
+            "oil": 42321,
+            "hydro": 78956,
+            "hydro storage": 21572,
+            "wind": 73393,
+            "solar": 13758,
+            "geothermal": 3811,
+            "other": 2040,
+            "_comment": "distributed PV: 9778MW, not included above"
```

Comment:
Can you move `_comment` one level up so it resides next to `capacity`?
I'm iterating through all keys in the `capacity` dictionary so it will create an error else :(